changelog.json

```diff
@@ -1,4 +1,59 @@
 [
+  {
+    "id": 7,
+    "slug": "enhanced-pipeline-orchestration-and-deployment-capabilities",
+    "title": "Enhanced Pipeline Orchestration and Deployment Capabilities",
+    "description": "ZenML now supports dynamic pipelines across local Docker and deployment scenarios, enabling more flexible workflow execution. The Kubernetes orchestrator includes graceful stopping for unhealthy steps via heartbeat monitoring, while the AzureML orchestrator and step operator now allow shared memory size configuration for better resource control.",
+    "published_at": "2025-12-16T09:19:32Z",
+    "published": true,
+    "audience": "oss",
+    "labels": [
+      "bugfix",
+      "improvement"
+    ],
+    "feature_image_url": "",
```

Comment:
We may want to add some illustration or so, so it doesnt fallback to the default images